src/features/packages/pack/zipUtils.ts

```diff
@@ -69,11 +69,12 @@ const setCompressionLevel = (zip: AdmZip, level: number): void => {
 
 async function expandGlobs(filePatterns: string[]): Promise<string[]> {
     const files: string[] = [];
+    const options: IOptions = { dot: true };
 
     for (const filePattern of filePatterns) {
         for (const fileName of filePattern.split(",")) {
             if (glob.hasMagic(fileName)) {
-                const filePaths = await globp(fileName);
+                const filePaths = await globp(fileName, options);

```

Comment:
Do we have any tests that can be updated to test that this does what we assume it does?

Reply:
not yet, just updating the create-zip action to test it manually first